qa-tests-backend/src/main/groovy/orchestratormanager/Kubernetes.groovy

```diff
@@ -634,7 +642,8 @@ class Kubernetes implements OrchestratorMain {
         return podsPassing == pods.size()
     }
 
-    def createJob(Job job) {
+    @CompileDynamic
```

Comment:
I suspect constructor with named args is a problem here but I don't know how to fix it :(